binary_transparency/firmware/cmd/publisher/publish.go

```diff
@@ -117,13 +122,28 @@ func main() {
 }
 
 func createManifestFromFlags() (api.FirmwareMetadata, []byte, error) {
+	var measure func([]byte) ([]byte, error)
+	switch *deviceID {
+	case "armory":
+		measure = usbarmory.ExpectedMeasurement
+	case "dummy":
+		measure = dummy.ExpectedMeasurement
+	default:
+		glog.Exitf("--device must be one of: 'dummy', 'armory'")
```

Comment:
This should return the error from here and `main` should Exit.

Reply:
done, ta